lib/screen_checker/gds_data/fetch.ex

```diff
@@ -132,23 +132,18 @@ defmodule ScreenChecker.GdsData.Fetch do
        }) do
     {screen_sn,
      %{
-       battery: parse_european_decimal(battery_str),
-       humidity: parse_european_decimal(humidity_str),
-       temperature: parse_european_decimal(temp_str),
-       log_time: parse_european_datetime(call_str),
+       battery: String.to_float(battery_str),
+       humidity: String.to_float(humidity_str),
+       temperature: String.to_float(temp_str),
+       log_time: parse_datetime(call_str),
        screen_name: screen_name,
        time: DateTime.utc_now(),
        ping_count: ping_count
      }}
   end
 
-  defp parse_european_decimal(s) do
-    [integer, decimal] = String.split(s, ",", parts: 2)
-    String.to_integer(integer) + String.to_float("0." <> decimal)
-  end
-
-  defp parse_european_datetime(s) do
-    with {:ok, naive_datetime} <- Timex.parse(s, "%d/%m/%Y %H:%M:%S", :strftime),
+  defp parse_datetime(s) do
+    with {:ok, naive_datetime} <- Timex.parse(s, "%-m/%-d/%Y %-I:%M:%S %p", :strftime),
          {:ok, dt} <- DateTime.from_naive(naive_datetime, "Europe/Rome"),
```

Comment:
Is this still the time zone their datetimes are expressed in? Seems odd that they'd change everything else to US conventions for our instance, but leave it on a European time zone.

Reply:
I also thought that was odd, but it does seem like the times are still in Rome's TZ, even though the formats changed.